src/tests/java/picard/illumina/parser/CycleIlluminaFileMapTest.java

```diff
@@ -75,42 +75,4 @@ private static String constructPathString(int lane, int cycle) {
             },
         };
     }
-
-  /*  @Test(dataProvider = "iteratorTestData")
```

Comment:
This file is currently doing nothing. I suspect that when originally written it was on it's way to being a test, but it seems that it become not important. I would vote for removing the whole file.